cmd/istioctl/main.go

```diff
@@ -309,13 +321,17 @@ func init() {
 }
 
 func main() {
+	if platform != kubePlatform {
```

Comment:
just dummy flag for now, issuing a warning when the default is changed (should probably map to an enum when really doing something)